TESTING.asciidoc

```diff
@@ -291,6 +292,20 @@ A specific test case can be run with
   -Dtests.method="test {p0=cat.shards/10_basic/Help}"
 ---------------------------------------------------------------------------
 
+To run REST tests of licensed features, use the following command:
+
+---------------------------------------------------------------------------
+./gradlew ':x-pack:plugin:integTestRunner' \
+  --tests "org.elasticsearch.xpack.test.rest.XPackRestIT"
+---------------------------------------------------------------------------
+
+A specific test case can be run with
+
+---------------------------------------------------------------------------
+./gradlew ':x-pack:plugin:integTestRunner' \
+  --tests "org.elasticsearch.xpack.test.rest.XPackRestIT.test {p0=indices.freeze/10_basic/Basic}"
```

Comment:
I believe this is one of those example that wont' actually work in this form because the test name includes a period (`.`). In that case you'd need to use the legacy `-Dtests.method` system property.